src/backend/aspen/api/conftest.py

```diff
@@ -143,7 +144,7 @@ async def auth0_oauth() -> MagicMock:
 
 @pytest.fixture()
 async def split_client() -> MagicMock:
-    return create_autospec(SplitClient)
+    return SplitClient(APISettings())
```

Comment:
Why do we want to use split's local functionality instead of mocking split's responses in our tests?

Reply:
so that we can pick up the values from `split.yml`, i felt like that was the most 'correct' way to go about this, do you have any potential concerns with this approach?

Reply:
Basically I want to make sure that we can test pathogen-specific functionality when we *truly* have features that only work for certain pathogens, but most of our tests should be able to do the right thing regardless of which pathogen we're working with. That probably means that we need to add repositories/prefixes to the test db when we're creating test pathogens

Reply:
ok i can change it back then